data/uking_functions.csv

```diff
@@ -73752,30 +73752,30 @@ Address,Quality,Size,Name
 0x0000007100d40828,O,000360,_ZN4ksys3map12PlacementMapC1Ev
 0x0000007100d40990,O,000644,_ZN4ksys3map12PlacementMapD1Ev
 0x0000007100d40c14,O,000072,_ZN4ksys3map12PlacementMap14loadStaticMap_Eb
-0x0000007100d40c5c,U,001692,PlacementMap::doLoadStaticMap_
+0x0000007100d40c5c,W,001692,_ZN4ksys3map12PlacementMap16doLoadStaticMap_Eb
```

Comment:
I think you may have accidentally committed changes from your PlacementMap branch?